### PR TITLE
feat(core,legacy): display only sat instead of sat BTC

### PR DESCRIPTION
**Pull request by marnova**

### CODE REVIEW

**Comment by matejcik** [comment]
there is no UI diff for this change.

does it mean that we don't have tests looking at `sat BTC`? If that is true, please make some :)

**Comment by matejcik** [comment]
you can take a simple signing test for the input data and add it to `test_signtx_amount_unit.py`, parametrized the same way the existing testcase that is already there.

File: tests/device_tests/bitcoin/test_signtx_amount_unit.py [hunk 1]
```diff
@@ -73,3 +82,50 @@ def test_signtx(client: Client, amount_unit):
...
+                request_input(0, TXHASH_0dac36),
+                request_output(0, TXHASH_0dac36),
+                request_output(1, TXHASH_0dac36),
+                request_input(0),
+                request_output(0),
+                request_output(0),
+                request_finished(),
+            ]
+        )
+
```
Comment (matejcik) [inline tests/device_tests/bitcoin/test_signtx_amount_unit.py:122]: please exclude this block, that's not what we are interested in here so it's just needlessly spamming the test size

**Comment by matejcik** [comment]
please fix style and squash everything. LGTM code-wise, just waiting for the UI test screens

**Review by matejcik — APPROVED** [review]
device tests are failing but they do show the difference that I was interested in, so :+1: on the result.

please merge when you get the CI green

File: tests/device_tests/bitcoin/test_signtx_amount_unit.py (hunk 1 above)
Comment (marnova) [reply tests/device_tests/bitcoin/test_signtx_amount_unit.py]: 0dede9b8d613477ea75203e19d1448945b6fd6ad

**Pull request merged**